tests/FSharp.Compiler.ComponentTests/Conformance/DelegateTypes/invalid_delegate_definition.fs

```diff
@@ -0,0 +1,2 @@
+﻿type T(x: int) =
```

Comment:
Please also add a test without any parameters, since an empty constructor is not allowed either.

Reply:
Updated following your suggestion